src/SUMMARY.md

```diff
@@ -6,6 +6,7 @@
     - [Build and Install distribution artifacts](./build-install-distribution-artifacts.md)
     - [Documenting Compiler](./compiler-documenting.md)
 - [Coding conventions](./conventions.md)
+- [Stabilizing Features](./stabilization_guide.md)
```

Comment:
Perhaps we could make this an appendix chapter?

Reply:
I meant that you could create a new one, i.e. make this a new appendix chapter. All of the appendix chapters are in the src/appendix/ directory, so you would just move this file to that directory and rename it if you wanted to make this an appendix.